description/description.rdfs.ttl

```diff
@@ -40,8 +40,8 @@ haDes:DVD   a   rdfs:Class ;
     rdfs:isDefinedBy <https://data.hetarchief.be/ns/description/> .
 
 haDes:DVDChapter   a   rdfs:Class ;
-    rdfs:label "Dvd-hoofdstuk"@nl ;
-    rdfs:label "DVD Chapter"@en ;
+    rdfs:label "DVD-hoofdstuk"@nl ;
```

Comment:
In het NL is het normaliter met kleine letters: https://woordenlijst.org/zoeken/?q=dvd
```suggestion
    rdfs:label "Dvd-hoofdstuk"@nl ;
```